repos/aspnetcore.proj

```diff
@@ -16,6 +16,10 @@
     <BuildCommandArgs>$(BuildCommandArgs) --arch $(Platform)</BuildCommandArgs>
     <BuildCommandArgs>$(BuildCommandArgs) /p:BuildNodeJs=false</BuildCommandArgs>
     <BuildCommandArgs>$(BuildCommandArgs) /p:SourceBuildRuntimeIdentifier=$(OverrideTargetRid)</BuildCommandArgs>
+    <BuildCommandArgs>$(BuildCommandArgs) /p:MicrosoftNetFrameworkReferenceAssembliesVersion=1.0.0</BuildCommandArgs>
+    <!-- aspnetcore requires some types in the RC1 version of this package.  This is temporary until RC1 versions
+        are included in previously source-built packages -->
+    <BuildCommandArgs>$(BuildCommandArgs) /p:MicrosoftNETCoreAppRefPackageVersion=5.0.0-rc.1.20451.14</BuildCommandArgs>
     <BuildCommandArgs>$(BuildCommandArgs) /p:UseAppHost=false</BuildCommandArgs>
```

Comment:
Add comment to the first one, put these two props after the rest of the args to stand them apart from the more typical build configuration changes?

Does the PVP not work for these?

Reply:
Done.  The PVP doesn't work because the PVP has the previously source-built version, which is a preview8 version.  When previously source-built is updated, this can be removed.